pkg/cmd/roachtest/cdc.go

```diff
@@ -41,13 +43,18 @@ func registerCDC(r *registry) {
 
 		c.Put(ctx, cockroach, "./cockroach", crdbNodes)
 		c.Put(ctx, workload, "./workload", workloadNode)
-		c.Start(ctx, crdbNodes)
+		c.Start(ctx, crdbNodes, startArgs(`--args=--vmodule=poller=2`))
 
 		t.Status("loading initial data")
 		c.Run(ctx, workloadNode, fmt.Sprintf(
 			`./workload fixtures load tpcc --warehouses=%d --checks=false {pgurl:1}`, warehouses,
 		))
 
+		// Clean up kafka if it was running to make development iteration
+		// easier.
+		c.Run(ctx, kafkaNode, `sudo killall -9 java | true`)
```

Comment:
Is there any chance of this having unexpected consequences? This seems heavyhanded. Perhaps this actually is the only java process running in our typical environments, but that might not always be the case for somebody's local machine - I'm not sure what the best approach here is.